pkg/mtbroker/filter/filter_handler.go

```diff
@@ -228,6 +228,7 @@ func (h *Handler) sendEvent(ctx context.Context, headers http.Header, target str
 	defer message.Finish(nil)
 
 	additionalHeaders := utils.PassThroughHeaders(headers)
+	kncloudevents.SetAcceptReplyHeader(additionalHeaders)
```

Comment:
I would remove this.

Reply:
Instead, add the header with a simple headers.Set() on line 197 of this file.

Reply:
IIUC `PassThroughHeaders` will filter unknown headers. I can also add the Prefer header to the pass through list and change it in 197.

Reply:
Does this alone cover the case of a channel is used for event delivery without mtbroker? My knowledge about the channel is limited, but I think this is allowed from the knative doc.

Reply:
then add `Prefer` to `PassThroughHeaders`